### ISSUE TITLE
Upload of gzipped gff3 files fail

### ISSUE THREAD

**Issue by lparsons** [galaxyproject/galaxy#9732]
Uploading or importing via URL a gff3 files fails. The logs show the following error:
```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```
Example file c_elegans.PRJNA275000.WS276.annotations.gff3.gz - ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA275000/c_elegans.PRJNA275000.WS276.annotations.gff3.gz

**Comment by lparsons** [comment]
Being addressed in #9731

**Comment by mvdbeek** [comment]
Should be fixed by https://github.com/galaxyproject/galaxy/pull/9731, thanks for the report @lparsons (and @jennaj).
